Make sure setuptools does not create a zip file in python_test; this is buggy
and causes tests to fail.

--- a/setup.py
+++ b/setup.py
@@ -179,6 +179,7 @@
                  ext_modules = ext_modules,
                  cmdclass = {'install_lib': MyInstallLib,
                              'build_py': MyBuildPy},
+                 zip_safe = False,
                  **kwargs
                  )
 
